Header header
geometry_msgs/Vector3[] data
float32[] bearing_rate
